### PR TITLE
Add file manager link to the top level

### PR DESCRIPTION
**Pull request by iliajie**

### CODE REVIEW

File: virtual-server-lib-funcs.pl [hunk 1]
```diff
@@ -12869,6 +12869,18 @@ sub get_domain_actions
 		    'icon' => 'page_edit',
 		  });
 	}
+
+if (&foreign_available("xterm") &&
+    !$d->{'alias'} && $d->{'user'}) {
+	# Link to Terminal for master admin
```
Comment (jcameron) [inline virtual-server-lib-funcs.pl:12875]: Should this be visible to the domain owner, or only the master admin? If the latter, you should add `&master_admin()` to the if statement.

Comment (iliajie) [reply virtual-server-lib-funcs.pl:12875]: It has to be shown to all users that have access to `xterm`, like File Manager.

It's convenient to have it shown easily accessible. 

image

Comment (jcameron) [reply virtual-server-lib-funcs.pl:12875]: Right but for other users, will that user= parameter work?

Comment (iliajie) [reply virtual-server-lib-funcs.pl:12875]: No, it will be ignored. Related - https://github.com/webmin/webmin/pull/1770/files

Comment (jcameron) [reply virtual-server-lib-funcs.pl:12875]: Ok, that's reasonable

**Pull request merged**